tutorial-5/src/server.rs

```diff
@@ -0,0 +1,43 @@
+use axum::{response::IntoResponse, routing::get, Router};
+use std::net::SocketAddr;
+use tracing::info;
+
+use crate::mockservices::{rust, RustService, Web3Service, WorldService};
+
+pub async fn start_server() {
+    // here we initialize logging
+    tracing_subscriber::fmt().init();
+
+    // here we create 3 instance
+    let rust_service = RustService::new();
+    let world_service = WorldService::new();
+    let web3_service = Web3Service::new();
+
+    // create a basic app of Axum with a handler to response 'Hello World'
+    let app = Router::new()
+        .route(
+            "/helloworld",
+            get(move || async { world_service.hello().await }),
```

Comment:
bind instances of structs' inner functions which with return value in type of `impl IntoResponse` as handlers to response request to specific endpoints.